apps/web/src/app/(signing)/sign/[token]/sign-dialog.tsx

```diff
@@ -23,7 +32,7 @@ export const SignDialog = ({
   onSignatureComplete,
 }: SignDialogProps) => {
   const [showDialog, setShowDialog] = useState(false);
-
+  const truncatedTitle = truncateTitle(document.title, 16);
```

Comment:
Out of curiosity, what is the reason behind `16` as the max length? Is it the maximum length without breaking the UI?

Reply:
@catalinpit Just a random length. I saw that it was correctly fitting for the current UI. It can be modified if needed.

Reply:
Got it!